### PR TITLE
Added code from sdx-transform-cs feature/process_CORA_destined_survey…

### PR DESCRIPTION
**Pull request by tundish**
…_data.

### CODE REVIEW

File: tests/pck/023.0102.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (necrophonic) [inline tests/pck/023.0102.json]: Don't need the `pck` files and tests - they're used by common software not cora

File: transform/surveys/023.0213.json [hunk 2]
```diff
@@ -0,0 +1,94 @@
+{
```
Comment (necrophonic) [inline transform/surveys/023.0213.json]: Don't need the mappings for non-ukis surveys here

File: transform/settings.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import logging
+import requests
+import os
+from requests.packages.urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter
+
+LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-transform-cs: %(message)s"
```
Comment (eldeal) [inline transform/settings.py:7]: -cora not -cs

File: transform/transformers/CSTransformer.py [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+        # self.create_pck()
+        self.create_idbr()
+        return path
+
+    def prepare_archive(self):
+        '''
+        Prepare a list of files to save
+        '''
+        # self.files_to_archive.append(("EDC_QData", self.pck_file))
```
Comment (necrophonic) [inline transform/transformers/CSTransformer.py:48]: Should this be commented out? If it's not required you can take it out.

**Pull request merged**